questionnaires/lipf/lipf_armt.json

```diff
@@ -644,14 +644,14 @@
         "text_validation_min": "",
         "text_validation_max": "",
         "identifier": "",
-        "branching_logic": "[lipf_cough_q1]!=0",
+        "branching_logic": "[lipf_cough_q1]=1 or [lipf_cough_q1]=2 or [lipf_cough_q1]=3 or [lipf_cough_q1]=4",
         "required_field": "",
         "custom_alignment": "",
         "question_number": "",
         "matrix_group_name": "lipf_cough",
         "matrix_ranking": "",
         "field_annotation": "range-type",
-        "evaluated_logic": "responses['lipf_cough_q1'] == 0"
+        "evaluated_logic": "responses['lipf_cough_q1'] == 1 || responses['lipf_cough_q1'] || responses['lipf_cough_q1'] || responses['lipf_cough_q1']"
```

Comment:
This is still not parsed correctly

Reply:
It's fixed now.

Reply:
Yes, I update in redcap to include the quotes. Thanks for pulling the updates.